core/remote/warning_poller.js

```
@@ -80,7 +80,6 @@ class RemoteWarningPoller {
       this.events.emit('remoteWarnings', warnings)
     } catch (err) {
       log.error({err})
-      throw err
```

Maybe we should swallow only some types of errors instead of all errors?